.github/workflows/ci.yaml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
   pull_request:
+  workflow_dispatch:
```

Comment:
This is useful for contributors. It allows them (and me) to run CI manually on specific branch on forks.

Reply:
Oh that's a really nice addition! 😃